pom.xml

```diff
@@ -18,28 +18,71 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.eclipse.microprofile.parent</groupId>
+    <groupId>org.eclipse.microprofile</groupId>
     <artifactId>microprofile-parent</artifactId>
-    <version>1-SNAPSHOT</version>
+    <version>1.0-SNAPSHOT</version>
+
     <packaging>pom</packaging>
-    <name>MicroProfile Parent</name>
-    <description>Parent POM for Eclipse MicroProfile</description>
+    <name>MicroProfile Parent POM</name>
     <url>http://microprofile.io</url>
 
     <properties>
+        <!-- Cross plugins settings -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- Compiler -->
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <checkstyle.version>2.17</checkstyle.version>
-        <checkstyle.methodNameFormat>^_?[a-z][a-zA-Z0-9]*$</checkstyle.methodNameFormat>
+
+        <!-- Dependencies - Jakarta -->
+        <dependency.jakarta.api.version>8.0.0</dependency.jakarta.api.version>
+        <dependency.jakarta.cdi.version>2.0.2</dependency.jakarta.cdi.version>
+        <dependency.jakarta.ws-rs.version>2.1.6</dependency.jakarta.ws-rs.version>
+        <dependency.jakarta.jsonp.version>1.1.6</dependency.jakarta.jsonp.version>
+        <dependency.jakarta.jsonb.version>1.0.2</dependency.jakarta.jsonb.version>
+        <!-- Dependencies - Test -->
+        <dependency.testng.version>6.14.3</dependency.testng.version> <!-- aligns with Arquillian TestNG version -->
+        <dependency.hamcrest.version>1.3</dependency.hamcrest.version>
+        <dependency.arquillian.version>1.6.0.Final</dependency.arquillian.version>
+
+        <!-- Plugins -->
+        <dependency.microprofile.build-tools>1.0-SNAPSHOT</dependency.microprofile.build-tools>
+
+        <plugin.resources.version>3.2.0</plugin.resources.version>
+        <plugin.jar.version>3.2.0</plugin.jar.version>
```

Comment:
I don't really care the difference between dependency and plugin. I suggest not to use different prefixes.